SampleShared/Samples/RenderingSample/Scripts/RenderingSettings.cs

```diff
@@ -1,23 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.OpenXR.Sample;
 using System;
 using System.Linq;
 using UnityEngine;
 using UnityEngine.XR.OpenXR;
 
 namespace Microsoft.MixedReality.OpenXR.Samples
 {
-    public class RenderingSettings : MonoBehaviour
+    public class RenderingSettings : MonoBehaviour, ITextProvider
     {
-        [SerializeField]
-        private TextMesh m_statusPanel;
+        private string m_text;
 
         private ReprojectionMode[] allReprojectionModes = (ReprojectionMode[])Enum.GetValues(typeof(Microsoft.MixedReality.OpenXR.ReprojectionMode));
         private ReprojectionMode targetReprojectionMode = ReprojectionMode.Depth;
-        private float m_stereoSeparationAdjustment = 0;
-        private bool m_stereoSeparationAdjustmentChanged = false;
+        public float m_stereoSeparationAdjustment { get; set; } = 0;
```

Comment:
By changing the property to "public" you are making this field to be serialized into unity scene.  Is this intentional?

BTW:  when the field is public, it should be named differently and don't use "m_" anymore.

Reply:
Yes this is intentional so that the slider can fill this field.